### ISSUE TITLE
Can't view zip files under MacOS 15.0

### ISSUE THREAD

**Issue by rhstanton** [mucommander/mucommander#1265]
### Is there an existing issue for this?

- [X] I have searched the existing issues (including the closed ones)

### Description

I just upgraded to MacOS 15.0 (not quite sure why...). muCommander works fine in general, but when I click on a zip file, instead of opening as usual, I get a box popping up saying "Password protected file?" and asking me to enter a password. It is NOT password protected.

### Steps to reproduce

Open muCommander under MacOS 15.0. Click on a zip file

### Expected behavior

Zip contents should be displayed in directory format

### Actual behavior

Box pops up asking me for password (zip file is not password protected)

### muCommander version

1.5.0

### Java version

Runtime version: 23-internal
VM name: OpenJDK 64-Bit Server VM
VM version: 23-internal-ahadas-0081ac197a656cdc0d88d5a5e2a0f399ee023654
VM vendor: Oracle Corporation

### Operating System type and version

Name: Mac OS X
Version: 15.0
Architecture: aarch64

### Relevant log output

```shell
[2024-09-17 16:38:27.091] SEVERE BrowseLocationThread.java#run,400 failed to browse location file://localhost/Users/stanton/Downloads/ttt.zip
```

**Comment by ShayArtzi** [comment]
The password dialog is being shown basically each time there is an error opening a zip file, under the assumption it is encrypted.

Couple of questions for you:

1. Are you able to upload the zip file in question so I can try it on my machine?
2. The file location seems a bit odd (`file://localhost`) - Just wanted to double check that the actual path is you local user directory.

**Comment by rhstanton** [comment]
Here's a sample zip file, but it doesn't seem to matter what file I use. This one I just created at the command line using 

"zip test test.txt"

Also, yes, the file is on my local hard drive inside my user directory. 
[test.zip](https://github.com/user-attachments/files/17036650/test.zip)

**Comment by ShayArtzi** [comment]
Okay, good news! apparently it's the same issue that was fixed in #1253. Do you mind trying out the nightly build and see if that resolves your issue? here is the link for mac os aarch64: https://github.com/mucommander/mucommander/actions/runs/10894447791/artifacts/1940394260

**Comment by rhstanton** [comment]
That seems to work. Thanks!

**Comment by ShayArtzi** [comment]
Glad to hear that - the fix should be included in the upcoming release (version 1.6.0)

**Comment by ahadas** [comment]
closing as a duplicate of #1252 
@ShayArtzi, about `1.6.0` mentioned above, I think we'll have 1.5.1 with this fix and other changes we did since 1.5.0 rather than waiting for a bigger release that would justify bumping the minor version

**Comment by ShayArtzi** [comment]
> closing as a duplicate of #1252 @ShayArtzi, about `1.6.0` mentioned above, I think we'll have 1.5.1 with this fix and other changes we did since 1.5.0 rather than waiting for a bigger release that would justify bumping the minor version

Sounds good :)
